### ISSUE TITLE
Is it possible to add two fragments in a fragment

### ISSUE THREAD

**Issue by ajmalsalim** [jfeinstein10/SlidingMenu#139]
As we have 2 main fragments, (Menu+Content), is it possible to add another 2 fragments in the Content Fragment while using ActionBarSherlock Tabs ?

**Comment by jfeinstein10** [comment]
No, it's not really possible to put Fragments in Fragments, but you can add multiple Fragments to a view by putting multiple containers in the view.

**Comment by ruiaraujo** [comment]
Now it is. Check the 4.2 APi changes. ;)

**Comment by jfeinstein10** [comment]
Haha why in the heck would they do that?! I'll look later :P

**Comment by ruiaraujo** [comment]
Flexibility. You're not forced to use it but it may come handy sometimes. :P

**Comment by staber** [comment]
fragception

seems odd to me too.

**Comment by ajmalsalim** [comment]
:( I got stuck 

Actually I have 2 Fragments, one is Menu and other is Content Fragment. Now I need to add Google Map to the Content Fragment. I have added Google Maps Plugin for ABS. But how to use it in a Fragment. I can 

public class MapActivity extends SherlockMapActivity 

but how to make it done in a Fragment

like: public class MapFragment extends SherlockMapFragment

Actually I want to use two tabs in a fragment (say Fragment C) and show some textview in one tab and mapview in other fragment. But  Fragment C is in Fragment B and Fragment A is SlidingMenu.

:(
